src/Service/Order/OrderService.php

```
@@ -0,0 +1,564 @@
+
+        // No need to map tokenizationMethod as it's already in camelCase in the API response
+        // But keep the mapping for other fields that might be in snake_case
+        if (isset($result['cardholder_name'])) {
+            $result['cardholderName'] = $result['cardholder_name'];
+        }
+        if (isset($result['cardholder_email'])) {
+            $result['cardholderEmail'] = $result['cardholder_email'];
+        }
+
+        return $result;
+    }
```

PHPMD (2.15.0)

Avoid unused private methods such as 'flattenPaymentMethodData'. (Unused Code Rules)

(UnusedPrivateMethod)